doc/rtd/topics/datasources/nocloud.rst

```diff
@@ -61,35 +61,42 @@ You may also optionally provide a vendor-data file in the following format.
 
   /vendor-data
 
-Given a disk ubuntu 12.04 cloud image in 'disk.img', you can create a
+Given a disk ubuntu cloud image in 'disk.img', you can create a
 sufficient disk by following the example below.
 
 ::
 
-    ## create user-data and meta-data files that will be used
+    ## 1) create user-data and meta-data files that will be used
     ## to modify image on first boot
     $ { echo instance-id: iid-local01; echo local-hostname: cloudimg; } > meta-data
-
     $ printf "#cloud-config\npassword: passw0rd\nchpasswd: { expire: False }\nssh_pwauth: True\n" > user-data
 
-    ## create a disk to attach with some user-data and meta-data
+    ## 2a) create a disk to attach with some user-data and meta-data
     $ genisoimage  -output seed.iso -volid cidata -joliet -rock user-data meta-data
 
-    ## alternatively, create a vfat filesystem with same files
+    ## 2b) alternatively, create a vfat filesystem with same files
     ## $ truncate --size 2M seed.img
     ## $ mkfs.vfat -n cidata seed.img
-    ## $ mcopy -oi seed.img user-data meta-data ::
 
-    ## create a new qcow image to boot, backed by your original image
-    $ qemu-img create -f qcow2 -b disk.img boot-disk.img
+    ## 2b) option 1: mount and copy files
+    ## $ sudo mount -t vfat /tmp/seed.img /mnt
+    ## $ sudo cp user-data meta-data /mnt
+    ## $ sudo umount /mnt
+
+    ## 2b) option 2: the mtools package provides mcopy, which can access vfat
+    filesystems without mounting them
+    ## $ mcopy -oi seed.img user-data meta-data
+
+    ## 3) create a new qcow image to boot, backed by your original image
+    $ qemu-img create -f qcow2 -b disk.img -F qcow2 -K qcow2 boot-disk.img
```

Comment:
`qemu-img: unrecognized option '-K'`

Even fixing that, I can't get this command to run.
```
qemu-img create -f qcow2 -b disk.img -F qcow2 boot-disk.img 
qemu-img: boot-disk.img: Could not open 'disk.img': No such file or directory
Could not open backing image.
```

I followed 2b option 1.

Reply:
The instructions start with:
```
Given a disk ubuntu cloud image in 'disk.img', you can create a sufficient disk by following the example below.
```

Reply:
Yes, I did. I suppose I should rename my image. :smile: